scripts/readme/readme.py

```diff
@@ -305,7 +305,8 @@ def write_readme(workflow_telemetries, readme_telemetries):
         "examples/tutorials/**/README.md",
         "examples/tools/use-cases/**/README.md"
     ]
+    input_glob_readme_exclude = ["examples/flows/integrations/**/README.md"]
```

Comment:
add comments why we need to exclude this.

Reply:
Okay